react-native/react/native/remote-manager.desktop.js

```diff
@@ -91,7 +91,6 @@ class RemoteManager extends Component {
             onRemoteClose={() => this.props.onCloseFromHeader(username)}
             component='tracker'
             username={username}
-            substore='tracker'
```

Comment:
Just be careful as calculating the diffs can be slower than just sending
the whole thing

On Monday, January 11, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In react-native/react/native/remote-manager.desktop.js
> <https://github.com/keybase/client/pull/1707#discussion_r49408719>:
>
> > @@ -91,7 +91,6 @@ class RemoteManager extends Component {
> >              onRemoteClose={() => this.props.onCloseFromHeader(username)}
> >              component='tracker'
> >              username={username}
> > -            substore='tracker'
>
> This was really tricky actually, since I was using state from the config
> substore and I had forgotten about this. I have another PR that lets us
> efficiently think about the whole store by just sending diffs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1707/files#r49408719>.
>